### ISSUE TITLE
Warnings about missing source map files when used in ParcelJS + TypeScript project

### ISSUE THREAD

**Issue by mdenburger** [ava/use-http#202]
When use-http is used in a [Parcel.js](https://parceljs.org/) + TypeScript project, the build logs several warnings "could not load source file X in source map of node_modules/use-http/dist/Y"

**To Reproduce**
Create a new Parcel app with TypeScript that uses use-http:

1. `yarn init -y`
2. `yarn add --dev parcel-bundler typescript`
3. add index.html:
```
<html>
<body>
  <script src="./index.ts"></script>
</body>
</html>
```
4. add index.ts:
```
import useFetch from "use-http";

console.log('test', useFetch);
```
5. Build and start:
`yarn run parcel index.html --open`

**Expected behavior**
Parcel does not log any warnings.

**Actual behavior**
Parcel logs the following warnings:
```
⚠️  Could not load source file "../src/index.ts" in source map of "node_modules/use-http/dist/index.js".
⚠️  Could not load source file "../src/useMutation.ts" in source map of "node_modules/use-http/dist/useMutation.js".
⚠️  Could not load source file "../src/useFetch.ts" in source map of "node_modules/use-http/dist/useFetch.js".
⚠️  Could not load source file "../src/useQuery.ts" in source map of "node_modules/use-http/dist/useQuery.js".
⚠️  Could not load source file "../src/Provider.tsx" in source map of "node_modules/use-http/dist/Provider.js".
⚠️  Could not load source file "../src/FetchContext.ts" in source map of "node_modules/use-http/dist/FetchContext.js".
⚠️  Could not load source file "../src/types.ts" in source map of "node_modules/use-http/dist/types.js".
⚠️  Could not load source file "../src/utils.ts" in source map of "node_modules/use-http/dist/utils.js".
⚠️  Could not load source file "../src/doFetchArgs.ts" in source map of "node_modules/use-http/dist/doFetchArgs.js".
⚠️  Could not load source file "../src/useFetchArgs.ts" in source map of "node_modules/use-http/dist/useFetchArgs.js".
⚠️  Could not load source file "../useSSR.ts" in source map of "node_modules/use-ssr/dist/useSSR.js".
⚠️  Could not load source file "../src/index.ts" in source map of "node_modules/use-http/dist/index.js".
⚠️  Could not load source file "../src/useFetch.ts" in source map of "node_modules/use-http/dist/useFetch.js".
⚠️  Could not load source file "../src/useMutation.ts" in source map of "node_modules/use-http/dist/useMutation.js".
⚠️  Could not load source file "../src/useQuery.ts" in source map of "node_modules/use-http/dist/useQuery.js".
⚠️  Could not load source file "../src/Provider.tsx" in source map of "node_modules/use-http/dist/Provider.js".
⚠️  Could not load source file "../src/FetchContext.ts" in source map of "node_modules/use-http/dist/FetchContext.js".
⚠️  Could not load source file "../src/types.ts" in source map of "node_modules/use-http/dist/types.js".
⚠️  Could not load source file "../src/utils.ts" in source map of "node_modules/use-http/dist/utils.js".
⚠️  Could not load source file "../src/useFetchArgs.ts" in source map of "node_modules/use-http/dist/useFetchArgs.js".
⚠️  Could not load source file "../src/doFetchArgs.ts" in source map of "node_modules/use-http/dist/doFetchArgs.js".
⚠️  Could not load source file "../useSSR.ts" in source map of "node_modules/use-ssr/dist/useSSR.js".
```

Several Parcel.js issues (e.g. https://github.com/parcel-bundler/parcel/issues/2356) [suggest](https://github.com/parcel-bundler/parcel/issues/2356#issuecomment-444814230) this is an issue of the published package.

**Comment by alex-cory** [comment]
Will take a look asap

**Comment by alex-cory** [comment]
It would be nice if you can't get this working in a codesandbox, maybe make a repo reproducing the bug ❤️

**Comment by mdenburger** [comment]
Sure, I've pushed a small demo project to https://github.com/mdenburger/use-http-202.

**Comment by alex-cory** [comment]
@mdenburger this should be fixed in `v0.4.5`
